styles/Notes.module.css

```diff
@@ -0,0 +1,80 @@
+.container {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  text-align: center;
+}
+
+.title {
+  font-size: 400%;
+  margin: 3rem 0rem 1rem 0rem;
+  font-weight: 900;
+  line-height: 2ex;
+}
+
+.titleHighlight {
+  color: #ff3f58;
+}
+
+.subtitle {
+  font-size: 200%;
+  font-weight: 500;
+}
+
+.noteCard {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  color: #fff;
+  margin: 2rem;
+  text-decoration: none;
+  transition: 300ms ease;
+}
+
+.noteCard:hover {
+  cursor: pointer;
+  transform: scale(1.1);
+}
+
+.titleDiv {
+  background-color: var(--theme-ui-colors-green);
```

Comment:
Is it possible to change this colour? I know that it's probably a placeholder for images (probably added in future commits), but it's important to have it still match the colour scheme.

Thoughts, @NebuDev14?

Reply:
Sure! I was just following the figma arav made haha

Reply:
Oh whoops, my bad then! 😝